composer/trainer/trainer.py

```diff
@@ -824,7 +824,7 @@ def __init__(
         # Save Checkpoint
         save_folder: Optional[str] = None,
         save_filename: str = 'ep{epoch}-ba{batch}-rank{rank}.pt',
-        save_latest_filename: Optional[str] = 'latest-rank{rank}.pt',
+        save_latest_filename: str = 'latest-rank{rank}.pt',
```

Comment:
Wait, isn't this actually optional? If none, won't save latest, right?

Reply:
oops... good save...